assets/scss/icons.scss

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('../fonts/icomoon.woff?plxcjd') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
```

Comment:
What's the purpose of this?

Reply:
This style sheet is given to you when you use icomoon. At the moment, we are not using classes, but at some point we will be, so this will need to be here. I can delete it for now if you wish.